src/controllers/home-page.controller.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {get} from '@loopback/openapi-v3';
+import * as fs from 'fs';
+import * as path from 'path';
+import {template, TemplateExecutor} from 'lodash';
+import {inject} from '@loopback/context';
+import {RestBindings, Response} from '@loopback/rest';
+
+const pkg = require('../../../package.json');
```

Comment:
From the perspective of clean design and Dependency Injection pattern, I think controller should not depend `package.json` directly, but get it injected from app context. Then the application should bind the content of `package.json` (maybe a subset of properties only?) in its constructor.

Not a big deal for this particular pull request, but I think we should think more about the design patterns when we start working on adding HomePageController to our project template.

Reply:
Yes, I'm thinking about the same direction while I work on this implementation.